packages/cyberstorm/src/index.ts

```diff
@@ -5,3 +5,4 @@ export * from "./components/MetaItem";
 export * from "./components/PackageFlag";
 export * from "./components/Tag";
 export * from "./components/PackageCard";
+export * from "./components/TextInput";
```

Comment:
Not sure what this branch is based on but the exports aren't in alphabetical order anymore?

Reply:
Well that's fine by me but if I was you I'd be mighty interested why lines keep changing places on their own. What if it was some other code, where the change would actually cause a bug? Might be worth investigating (although I'm not sure where one would start such an investigation).